requirements-dev.txt

```diff
@@ -6,3 +6,5 @@ pytest-cython
 restructuredtext-lint
 sphinx
 sphinx_rtd_theme
+nose
```

Comment:
That's fine - if the sklearn tests weren't causing so many issues I would argue for including nose, but since I don't want to spend a ton of time debugging Python 3.4 pickling errors I took them out for now. I can say that the sklearn interface tests were passing at least in 3.5, so the current code meets all the primary BaseEstimator requirements.